src/test/datascience/uiTests/helpers.ts

```diff
@@ -141,6 +141,14 @@ export class BaseWebUI implements IAsyncDisposable {
         await this.browser.newContext();
         this.page = await this.browser.newPage();
         await this.page.goto(url);
+        this.page.on('console', (output) => {
+            try {
+                // Log output for diagnostic purposes (see what errors & the like are printed in console).
+                console.info(`IPyWidgets Browser: ${output.type()} ${output.text()}`);
```

Comment:
@joyceerhl added some logging so we can see what's going on in the browser side (i.e look at errors & the like logged to the browser console).
Hopefully useful for diagnosing issues (can remove later when we have addressed flakyness of ipywidget tests)